spec/controllers/posts_controller_spec.rb

```diff
@@ -1052,6 +1052,85 @@
       end
     end
 
+    it "gives the correct previous post" do
```

Comment:
*with an intermediate private post

Reply:
Do we test it without an intermediate private post, actually? I can't see a test for that – it might be worth adding one.